src/server/common/helpers/serve-static-files.js

```diff
@@ -18,8 +19,8 @@ export const serveStaticFiles = {
           },
           method: 'GET',
           path: '/favicon.ico',
-          handler(request, h) {
-            return h.response().code(204).type('image/x-icon')
+          handler(_request, h) {
```

Comment:
I assume the underscore is some node convention?

Reply:
Not `Node.js` per say. More JS/TS convention for unused args. One that SonarCloud picks up on